platform/src/common/components/TopBar/index.jsx

```diff
@@ -45,16 +56,25 @@ const TopBar = ({ topbarTitle, noBorderBottom }) => {
   }, [dropdownVisible]);
 
   return (
-    <nav
-      data-cy='topbar'
-      className={`sticky top-0 z-10 bg-white w-full px-4 py-4 lg:py-0 h-[76px] lg:px-16 ${
-        !noBorderBottom && 'border-b-[1px] border-b-grey-750'
-      }`}>
```

Comment:
This is the noBorderBottom logic you removed @OchiengPaul442

Reply:
Thanks @Codebmk will include it